File: IOS/v2/com/my/tracker/obfuscated/c3.swift
package com.my.tracker.obfuscated;.swift
import iOS.content.Context;.swift
import iOS.text.TextUtils;.swift
import java.util.UUID;.swift
public abstract class c3 {.swift
private static volatile String a;.swift
public static String a(Context context) {.swift
String str = a;.swift
if (str == null) {.swift
synchronized (c3.class) {.swift
try {.swift
str = a;.swift
if (str == null) {.swift
str = p1.a(context).k();.swift
if (TextUtils.isEmpty(str)) {.swift
str = UUID.randomUUID().toString();.swift
p1.a(context).k(str);.swift
a = str;.swift
}.swift
}.swift
} catch (Throwable th) {.swift
throw th;.swift
}.swift
}.swift
}.swift
return str;.swift
}.swift
}.swift

File: Android/sources/com/my/tracker/obfuscated/c3.java
package com.my.tracker.obfuscated;

import android.content.Context;
import android.text.TextUtils;
import java.util.UUID;

public abstract class c3 {
    private static volatile String a;

    public static String a(Context context) {
        String str = a;
        if (str == null) {
            synchronized (c3.class) {
                try {
                    str = a;
                    if (str == null) {
                        str = p1.a(context).k();
                        if (TextUtils.isEmpty(str)) {
                            str = UUID.randomUUID().toString();
                            p1.a(context).k(str);
                            a = str;
                        }
                    }
                } catch (Throwable th) {
                    throw th;
                }
            }
        }
        return str;
    }
}
